jaxrs/pom.xml

```diff
@@ -138,6 +114,30 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Let's not move these dependencies. The comment above "used by tests but also needed transitively" is only true for `json` but not for these. We could revisit this later if we apply the POM order enforcement.

Reply:
Sure. Restored previous order. I'll follow up with the pedantic pom enforcer